grammar/Expr.g4

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright (c) 2024, RTE (https://www.rte-france.com)
+
+See AUTHORS.txt
+
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+SPDX-License-Identifier: MPL-2.0
+
+This file is part of the Antares project.
+*/
+
+grammar Expr;
+
+/* To match the whole input */
+fullexpr: expr EOF;
+
+shift: TIME (op=('+' | '-') expr)?;
+
+expr: '-' expr                  # negation
+    | expr op=('/' | '*') expr  # muldiv
+    | expr op=('+' | '-') expr  # addsub
+    | expr COMPARISON expr      # comparison
+    | IDENTIFIER                # identifier
+    | IDENTIFIER '.' IDENTIFIER # portField
```

Comment:
Several things to answer this issue:
1. The parser operates in 2 phases, first identifying tokens (lexer), then only apply grammar rules (parser), see https://media.pragprog.com/titles/tpantlr2/picture.pdf. This parser rule cannot apply to floating point numbers, because we are here in the parser phase, where tokens have already been identified.
2. In the 1st phase, the lexer phase, one of the tokens definitions is that IDENTIFIER cannot start with a number. So for example 2.36 cannot be identified as IDENTIFIER.IDENTIFIER, because 2 is not a valid identifier.

Therefore there is no ambiguity here !